src/deployment/registration.py

```diff
@@ -369,13 +414,32 @@ def main():
     logging.basicConfig(format="%(levelname)s:%(message)s", level=level)
     logging.getLogger("deploy").setLevel(logging.INFO)
 
+    onefuzz_instance_name = args.onefuzz_instance
     if args.command == "update_pool_registration":
-        update_pool_registration(args.onefuzz_instance)
+        registration_name = "%s_pool" % onefuzz_instance_name
+        create_and_display_registration(
+            onefuzz_instance_name, registration_name, OnefuzzAppRole.ManagedNode
+        )
+    elif args.command == "create_cli_registration":
+        registration_name = onefuzz_instance_name or ("%s_cli" % onefuzz_instance_name)
+        create_and_display_registration(
+            onefuzz_instance_name, registration_name, OnefuzzAppRole.ManagedNode
+        )
     elif args.command == "assign_scaleset_role":
-        assign_scaleset_role(args.onefuzz_instance, args.scaleset_name)
+        assign_scaleset_role(onefuzz_instance_name, args.scaleset_name)
     else:
         raise Exception("invalid arguments")
 
 
 if __name__ == "__main__":
     main()
+
+# todo: update the internal docs (https://dev.azure.com/msresearch/OneFuzz/_git/OneFuzz?path=%2FREADME.md&_a=preview&version=GBinternal-information) with
+
+# Mandatory role assigment
+# if your org require that you prevent guest access by requiring a role assigment.
+# The default CLI will not be able to access the service.
+# you will need to create a new cli registration by executing the following
+# ```
+# python registration.py create_cli_registration INSTANCENAME [--registration_name REGISTRATIONNAME]
+# ```
```

Comment:
Reminder to update the documentation

Reply:
will update the docs once this PR is approved 

### Guest account access
If you need [prevent guest access](https://osgwiki.com/wiki/Guest_Accounts_in_Azure#2._Prevent_users_not_assigned_to_your_app_from_accessing_your_application.2Fservice) by requiring a role assigment, the default CLI will not be able to access the service.
You will need to create a new cli registration by executing the following.
```
python registration.py create_cli_registration INSTANCENAME [--registration_name REGISTRATIONNAME]
```

Then use the provided credentials in your onefuzz config 
```
onefuzz config --client_id CLIENTID
```